includes/class-sensei-frontend.php

```diff
@@ -1732,7 +1732,7 @@ public function sensei_process_registration(){
 		}
 
 		// notify the user
-		wp_new_user_notification( $user_id, $new_user_password );
+		wp_new_user_notification( $user_id, null, 'user' );
```

Comment:
Can we do a version check as we support WP >= 4.1 and the deprecation was after 4.3.1